Sources/PostgresNIO/New/PSQLCodable.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 11.01.21.
+//
+
+/// A type that can encode itself to a postgres wire binary representation.
```

Comment:
Ditch Xcode template headers.
```suggestion
/// A type that can encode itself to a postgres wire binary representation.
```